opal/corelib/comparable.rb

```diff
@@ -80,7 +80,7 @@ def between?(min, max)
     true
   end
 
-  def clamp(min, max=nil)
```

Comment:
Can you please squash the rubocop fixes to the initial commit in which the change was introduced?